src/app/qgisapp.h

```diff
@@ -916,6 +916,20 @@ class APP_EXPORT QgisApp : public QMainWindow, private Ui::MainWindow
     //! Update project menu with the project templates
     void updateProjectFromTemplates();
 
+    /**
+     * Settings pages section
+     */
+    //! Get map of option pages
+    QMap< QString, QString > *getOptionsPagesMap();
```

Comment:
No need for return by pointers here - QMaps are implicitly shared, so returning a copy is almost no-cost (and easier to manage!)

Reply:
QGIS convention is not to name getters with a "get" prefix. -> `optionsPagesMap()`